dht/token.hh

```diff
@@ -65,12 +68,20 @@ public:
     //     [0x80] == 0.5
     //     [0x00, 0x80] == 1/512
     //     [0xff, 0x80] == 1 - 1/512
-    managed_bytes _data;
+    std::array<uint8_t, 8> _data;
 
     token() : _kind(kind::before_all_keys) {
     }
 
-    token(kind k, managed_bytes d) : _kind(std::move(k)), _data(std::move(d)) {
+    token(kind k, std::array<uint8_t, 8> d) : _kind(std::move(k)), _data(std::move(d)) {
```

Comment:
Note that std::move()ing std::array is copying it. No harm done though.

Reply:
I removed move and took d by const& to avoid one copy.